app/src/main/java/ch/epfl/sdp/lobby/global/GlobalLobbyAdapter.kt

```diff
@@ -20,7 +20,7 @@ class GlobalLobbyAdapter(var data: List<Game>) :  RecyclerView.Adapter<MyViewHol
         override fun onClick(v: View?) {
             val game = data[layoutPosition]
             Toast.makeText(v?.context, game.name, Toast.LENGTH_SHORT).show()
-            val intent = Intent(v?.context, LobbyActivity::class.java) //TODO: Call with the game as param
+            val intent = Intent(v?.context, GameLobbyActivity::class.java) //TODO: Call with the game as param
```

Comment:
This will break "GlobalLobbyFragmentTest" line 105:         `intended(IntentMatchers.hasComponent(LobbyActivity::class.java.name))`
Please change it to 
`        intended(IntentMatchers.hasComponent(GameLobbyActivity::class.java.name))`